cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -131,7 +131,7 @@ public BuildCommand(Path projectPath, PrintStream outStream, PrintStream errStre
     private String debugPort;
 
     private static final String buildCmd = "ballerina build [-o <output>] [--offline] [--skip-tests]\n" +
-            "                    [<ballerina-file | ballerina-project>] [(--key=value)...]";
+            "                    [<ballerina-file | project-path>] [(--key=value)...]";
```

Comment:
It should be `package-path` IMO. Shall we remove the usage of the term project?

Reply:
There is a section as `Project Commands` when we run the `ballerina help` command. Should we change it to `Package Commands` here as well?